Units/parser-eiffel.r/obsolete.e.d/input.e

```diff
@@ -3,6 +3,8 @@ class OBSOLETE_ obsolete "Use a different one"
 feature
 
 	obsolete_attribute: INTEGER obsolete "Use something else"
```

Comment:
Yes, without the "attribute...end" it's not valid Eiffel syntax though the Ctags parser handles it.

I assume I missed it the first time around when eyeballing it due to the the unusual (for Eiffel) one-line formatting.

I'm 'linting' the input files now after changes since my skills as the human syntax parser are obviously on the wane.